first-bank-of-change/docker-compose.yaml

```diff
@@ -0,0 +1,24 @@
+version: '3'
+
+networks:
+  bdd:
+
+services:
+  npm:
+    networks:
+      - bdd
+    build:
+      dockerfile: npm.Dockerfile
+    volumes:
+      - .:/var/opt/app
+    working_dir: /var/opt/app
+    ports:
+      - "4200:4200"
+  selenium:
+    networks:
+      - bdd
+    image: seleniarm/standalone-chromium:114.0
```

Comment:
Need to test the [x86 image](https://hub.docker.com/layers/selenium/standalone-chrome/114.0/images/sha256-1b809a961a0a77787a7cccac74ddc5570b7e89747f925b8469ddb9a6624d4ece?context=explore) on a non-Apple Silicon system

Reply:
Actually, the one above works on x86 too!